relays/bin-substrate/src/cli/send_message.rs

```diff
@@ -110,6 +116,10 @@ impl SendMessage {
 
 			encode_call::preprocess_call::<Source, Target>(message, bridge.bridge_instance_index());
 			let target_call = Target::encode_call(message)?;
+			let target_spec_version = self
```

Comment:
That's the main change - instead of always using `Target::RUNTIME_VERSION.spec_version`, I'm using version based on CLI flags